classes/class-admin.php

```diff
@@ -687,6 +690,94 @@ public function render_list_table() {
 	<?php
 	}
 
+	public function maybe_render_csv_page() {
```

Comment:
I would suggest moving the `maybe_render_csv_page`, `render_csv_disable_paginate`, and `render_csv_expand_columns` into their own `Export` class, then instantiating it as a property of `Admin`.